website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -566,6 +637,53 @@ with events as (
 ... rest of model ...
 ```
 
+<VersionBlock firstVersion="1.4">
+
+#### Copying ingestion-time partitions
+
+If you have configured your incremental model to use "ingestion"-based partitioning (`partition_by.time_ingestion_partitioning: True`), you can opt to use a legacy mechanism for inserting and overwriting partitions. While this mechanism doesn't offer the same visibility and ease of debugging as the SQL `merge` statement, it can yield significant savings in time and cost for large datasets. Behind the scenes, dbt will add or replace each partition via the [copy table API](https://cloud.google.com/bigquery/docs/managing-tables#copy-table) and partition decorators.
+
+You can enable this by switching on `copy_partitions: True` in the `partition_by` configuration. This approach works only in combination with "dynamic" partition replacement.
```

Comment:
AFAIK there is no limitation that involves using only "dynamic" partitions, it might worth adding a test to confirm it though
https://getdbt.slack.com/archives/C99SNSRTK/p1674708308209719?thread_ts=1674124528.527979&cid=C99SNSRTK